scm/build/build.sh

```diff
@@ -1,15 +1,26 @@
 #!/bin/sh -ex
 export GOOS=linux
 cd $GOPATH/src/github.com/nokia/danm
-glide install --strip-vendor
-go get -d github.com/vishvananda/netlink
-go get github.com/containernetworking/plugins/pkg/ns
-go get github.com/golang/groupcache/lru
-LATEST_TAG=$(git describe --tags)
-COMMIT_HASH=$(git rev-parse HEAD)
+
+#
+# If we're being invoked inside Docker by the /build_danm script, use the
+# COMMIT_HASH and LATEST_TAG values that were already set by the invoking script.
+# Oherwise (eg if build.sh invoked directly during development cycle), set them
+# here.
+#
+if [ -z "${COMMIT_HASH}" ]
+then
+  COMMIT_HASH=$(git rev-parse HEAD)
+fi
+if [ -z "${LATEST_TAG}" ]
+then
+  LATEST_TAG=$(git describe --tags)
+fi
+
 go install -a -ldflags "-extldflags '-static' -X main.version=${LATEST_TAG} -X main.commitHash=${COMMIT_HASH}" github.com/nokia/danm/cmd/danm
 go install -a -ldflags "-extldflags '-static' -X main.version=${LATEST_TAG} -X main.commitHash=${COMMIT_HASH}" github.com/nokia/danm/cmd/netwatcher
 go install -a -ldflags "-extldflags '-static' -X main.version=${LATEST_TAG} -X main.commitHash=${COMMIT_HASH}" github.com/nokia/danm/cmd/svcwatcher
 go install -a -ldflags "-extldflags '-static' -X main.version=${LATEST_TAG} -X main.commitHash=${COMMIT_HASH}" github.com/nokia/danm/cmd/webhook
 go install -a -ldflags "-extldflags '-static'" github.com/nokia/danm/cmd/fakeipam
-go install -a -ldflags "-extldflags '-static'" github.com/nokia/danm/cmd/cnitest
\ No newline at end of file
+go install -a -ldflags "-extldflags '-static'" github.com/nokia/danm/cmd/cnitest
+go install -a -ldflags "-extldflags '-static'" github.com/containernetworking/plugins/plugins/main/macvlan
```

Comment:
ah okay, so you do meant macvlan :D
then maybe it is just a documentation mismatch, but the two are not the same IMO (e.g. https://github.com/maiqueb/macvtap-cni , and this is a "normal" bridged network interface whereas macvlan creates macvlan slaves directly attached to the host device)